easybuild/easyconfigs/v/VMTK/VMTK-1.4.0-foss-2019b-Python-3.7.4.eb

```diff
@@ -0,0 +1,87 @@
+# This file is an EasyBuild reciPY as per https://github.com/easybuilders/easybuild
+# Author: Denis Kristak (INUITS)
+
+easyblock = "CMakeMake"
+
+name = 'VMTK'
+version = '1.4.0'
+
+homepage = 'https://vmtk.org'
+description = """vmtk is a collection of libraries and tools
+for 3D reconstruction, geometric analysis, mesh generation and
+surface data analysis for image-based modeling of blood vessels. """
+
+toolchain = {'name': 'foss', 'version': '2019b'}
+versionsuffix = '-Python-%(pyver)s'
+local_vtk_shortver = '8.2'
+local_itk_shortver = '4.13'
+
+# fetching specific commit of VMTK bcs of some bugfixes
+local_commit = '63a3b99'
+
+local_vmtk_test_commit = "cbf09e4da03aa332c584b2709c4254017db9d4ef"
+source_urls = [
+    'https://github.com/vmtk/vmtk/archive/',
+    "https://raw.githubusercontent.com/vmtk/vmtk-test-data/%s/input/" % local_vmtk_test_commit,
+]
+sources = [
+    {'download_filename': '%s.tar.gz' % local_commit, 'filename': SOURCE_TAR_GZ},
+    {'filename': 'vase.vti', 'extract_cmd': 'cp %s %(builddir)s'},
+]
+checksums = [
+    '39aeb1de6bba8511dff45f0a69c9d6dc8d61f323739312ddceb22f8773e21459',  # VMTK-1.4.0.tar.gz
+    'b5f761d7a923ee19cd9cc8df7fbc794ffaa16aca101455fcdf9ceaa610ca3f57',  # vase.vti
```

Comment:
please change comment to `vase-20180313.vti`

Reply:
done